CHANGELOG.rst

```diff
@@ -4,6 +4,20 @@
 Synapse Changelog
 *****************
 
+Announcement
+============
+
+Due to the introduction of several powerful new APIs and performance
+improvments, Synapse will be updating to *only* support Python >=3.11.
+Our current plan is to drop support for Python <= 3.10 in ~4 weeks on
```

Comment:
this line has `<= 3.10` with a space in it and the previous line has `>=3.11` without a space in it, should make it consistent.